api-report/sequence.api.md

```diff
@@ -136,13 +136,28 @@ export interface IInterval {
 export interface IIntervalCollection<TInterval extends ISerializableInterval> extends TypedEventEmitter<IIntervalCollectionEvent<TInterval>> {
     // (undocumented)
     [Symbol.iterator](): Iterator<TInterval>;
+    // @deprecated
     add(start: number, end: number, intervalType: IntervalType, props?: PropertySet, stickiness?: IntervalStickiness): TInterval;
+    add({ start, end, props, stickiness, }: {
+        start: number;
+        end: number;
+        props?: PropertySet;
+        stickiness?: IntervalStickiness;
+    }): TInterval;
     // (undocumented)
     attachDeserializer(onDeserialize: DeserializeCallback): void;
     // (undocumented)
     readonly attached: boolean;
     attachIndex(index: IntervalIndex<TInterval>): void;
+    // @deprecated
     change(id: string, start?: number, end?: number): TInterval | undefined;
+    change({ id, start, end, props, }: {
```

Comment:
IMO it'd be best to make this `change(id, { start, end, props })` rather than put ID in the object. This would mimic how you might change a value within a data structure that supports lookup by some key.